pkg/loader/util.go

```diff
@@ -164,7 +167,8 @@ func createDBWitSessions(dsn string, params map[string]string) (db *gosql.DB, er
 	}
 
 	for k, v := range support {
-		dsn += fmt.Sprintf("&%s=%s", k, url.QueryEscape(v))
+		// For key='value', the value should be quoted to prevent it from being regarded as a key word
```

Comment:
https://github.com/go-sql-driver/mysql#system-variables

you can see the example `time_zone=%27Europe%2FParis%27: SET time_zone='Europe/Paris'`